cmake/DaemonGame.cmake

```diff
@@ -84,12 +85,13 @@ function(GAMEMODULE)
                     -DDAEMON_DIR=${Daemon_SOURCE_DIR}
                     -DDEPS_DIR=${DEPS_DIR}
                     -DBUILD_GAME_NACL_NEXE=${BUILD_GAME_NACL_NEXE}
-                    -DBUILD_GAME_NACL=1
-                    -DBUILD_GAME_NATIVE_DLL=0
-                    -DBUILD_GAME_NATIVE_EXE=0
-                    -DBUILD_CLIENT=0
-                    -DBUILD_TTY_CLIENT=0
-                    -DBUILD_SERVER=0
+                    -DBUILD_GAME_NACL_NEXE_ARM=${BUILD_GAME_NACL_NEXE_ARM}
+                    -DBUILD_GAME_NACL=ON
```

Comment:
Please avoid additional random cosmetic changes in this mega commit.

Reply:
Moved to dedicated bikeshedding commit.